src/LearningObjectStats/LearningObjectStatStore.ts

```diff
@@ -5,6 +5,7 @@ import {
 import { Db } from 'mongodb';
 import { COLLECTIONS } from '../drivers/MongoDriver';
 
+const ONION_BLOOMS= 'blooms_outcome_distribution'
```

Comment:
To make the name here more descriptive of what the string value I'd suggest changing it to something along the lines of
```suggestion
const BLOOMS_DISTRIBUTION_COLLECTION= 'blooms_outcome_distribution'
```